### PR TITLE
Got the getCountyAnnotationsCmd finished and manually tested

### PR DESCRIPTION
**Pull request by wrwalker**

### CODE REVIEW

File: SQLClient/src/internal/queryMaker/queryMaker.go [hunk 1]
```diff
@@ -82,6 +82,30 @@ func (q *QueryMaker) GetVotesForCandidate(candidate, county string, states []str
...
+}
+
+func (q *QueryMaker) GetCountyAnnotations(state string, county string) ([]map[string]interface{}, []string, string, error) {
+	query := getStringForGetCountyAnnotations(state, county)
+	rows, colNames, err := q.DoRawQuery(query)
+	if err != nil {
+		return nil, nil, "", err
+	}
+	if len(rows) < 1 || rows[0][colNames[0]] == nil {
+		return nil, nil, "", errors.New("could not find any matches")
```
Comment (Jspsun) [inline SQLClient/src/internal/queryMaker/queryMaker.go]: this should still return the string query so we can log it from the caller

Comment (wrwalker) [reply SQLClient/src/internal/queryMaker/queryMaker.go]: LOL this is literally one of the 2 bugs I fixed earlier and I just added another one. Good catch LOL tyty

Comment (wrwalker) [reply SQLClient/src/internal/queryMaker/queryMaker.go]: You can notice on this branch that the same bug is in GetVotesForCandidate which I mindlessly copied which is why its here as well LOL

File: SQLClient/src/internal/queryMaker/queryMaker.go [hunk 2]
```diff
@@ -82,6 +82,30 @@ func (q *QueryMaker) GetVotesForCandidate(candidate, county string, states []str
...
+		qString = fmt.Sprintf("%s and county=%q", qString, county)
+	}
+	return qString
+}
+
+func (q *QueryMaker) GetCountyAnnotations(state string, county string) ([]map[string]interface{}, []string, string, error) {
+	query := getStringForGetCountyAnnotations(state, county)
+	rows, colNames, err := q.DoRawQuery(query)
+	if err != nil {
+		return nil, nil, query, err
```
Comment (wrwalker) [inline SQLClient/src/internal/queryMaker/queryMaker.go:101]: Added fix. Thanks for the catch.

**Pull request merged**